packages/common/src/components/Table/ExpandableTableRow.tsx

```diff
@@ -1,13 +1,19 @@
 import ArrowToggle from '@frontend/common/src/components/utils/ArrowToggle'
 import React, { Fragment, useState } from 'react'
+import styled from 'styled-components'
+
+const StyledButton = styled.button<{ disabled: boolean }>`
+  cursor: ${(props) => (props.disabled ? 'default' : 'pointer')};
```

Comment:
Is this necessary when the button has the attribute disabled, shouldn't it come automatically?

Reply:
It didn't come automatically, that's why I added that css.

Reply:
The default cursor for a disabled button is a pointer so I don't think you need to add logic for which cursor should be applied. If you have added som classnames which overrides the cursor: default styling then that should be the only thing you need to add.

Reply:
Ah, I removed the logic and added some classes instead now.